examples/plot_model_card.py

```diff
@@ -60,49 +61,64 @@
 # Create a model card
 # ====================
 # We now create a model card, set couple of attributes and save it.
```

Comment:
This sentence conflicts with the next one, the next one makes it feel like user saves it then adds data